src/main/java/com/uploadcare/api/Image.java

```diff
@@ -0,0 +1,118 @@
+package com.uploadcare.api;
+
+import com.uploadcare.data.FileData;
+import com.uploadcare.data.ImageInfo;
+
+import java.util.Date;
+
+/**
+ * Created by Egor Petushkov
+ * on 09.06.17.
+ */
+public class Image extends File {
+    public static final int UNKNOW_DPI = -1;
+    public enum Orientation {
```

Comment:
I don't think that this adds any value vs using numbers.
That's why you need those lengthy comments :)
Can you please elaborate to defend this?